composer.json

```diff
@@ -25,7 +25,7 @@
     "docs": "https://docs.typo3.org/p/kitodo/presentation/main/en-us/"
   },
   "require": {
-    "php": "7.4 - 8.3",
+    "php": "8.1 - 8.4",
```

Comment:
```suggestion
    "php": "8.1 - 8.3",
```

Reply:
Why did you change this back to `8.4`? Official support should be for the same PHP versions as TYPO3 v11 and v12, so PHP 8.1, 8.2 and 8.4.
You can choose to use another PHP version, but that is up to you und should not be reflected in the official `composer.json`. See https://getcomposer.org/doc/06-config.md#platform for how to set Composer to use a different PHP version, but resolve dependencies anyway.